services/workflows-service/src/common/utils/random/index.ts

```diff
@@ -0,0 +1,35 @@
+const SYMBOLS = '!@#$%^&*()_+-=[]{}|;:,.<>?';
+const NUMBERS = '0123456789';
+const CHARS = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz';
+
+type Options = {
+  includeSymbols?: boolean;
+  includeNumbers?: boolean;
+  includeAlphabetics?: boolean;
+};
+
+const defaultOptions: Options = {
+  includeSymbols: false,
+  includeNumbers: true,
+  includeAlphabetics: true,
+};
+
+export const generateRandomId = (length: number, options?: Options) => {
```

Comment:
why do you need this?
you have uuid

Reply:
just shorter id and supported cross platform compared to crypto library